pkg/rootless/rootless_linux.c

```diff
@@ -200,6 +200,9 @@ can_use_shortcut ()
   if (argv == NULL)
     return false;
 
+  if (strstr (argv[0], "podman") == NULL)
```

Comment:
What happens if I execute /usr/bin/podman?  Should this me just checking for paths ending in /podman or podman?

Reply:
@rhatdan strstr() should only return NULL if "podman" is not contained in argv[0]

Reply:
No that is fine. My lying eyes saw strcmp now I see strstr...

Reply:
yes as long as podman is in the string then the process tries to join the user namespace.

I need to refactor this part and move the podman specific bits our of the rootless package